src/main/kotlin/app/Config.kt

```diff
@@ -13,4 +13,7 @@ object Config {
     // Get 'unrestricted' state from Heroku/System, or return null if not set
     fun getUnrestrictedState(): String? = ProcessBuilder().environment()["UNRESTRICTED"] ?: System.getProperty("unrestricted")
 
+    // get 'tracking' state from Heroku/System, or return null if not set
+    fun getTrackingState(): String? = ProcessBuilder().environment()["TRACKING"] ?: System.getProperty("tracking")
```

Comment:
Toggling GTM as a `tracking` boolean doesn't make sense, as the choice becomes `no tracking` or `tipsy's tracking` (which no one can use).

Instead it should be `gtmId`, and `gtm.html` should be turned into a `.vm`, where `GTM-W4XBCK` is replaced with `$gtmId`. This part also needs to be explained in the readme.